hw/fpga/scripts/xilinx_generate_clk_wizard.tcl

```diff
@@ -0,0 +1,50 @@
+
+# Define design macros
+set design_name                 xilinx_clk_wizard
+set in_clk_freq_hz              125000000
+set out_clk_freq_hz             20000000
+set clkin1_jitter_ps            80.0
+set clkout1_jitter              172.798
+set clkout1_phase_error         96.948
+set clkout1_requested_out_freq  20
+set mmcm_clkfbout_mult_f        8.000
+set mmcm_clkin1_period          8.000
+set mmcm_clkout0_divide_f       50.000
+set prim_in_freq                125.000
+set use_locked                  false
```

Comment:
we do not need this value as a variable, please use it directly in the creation of the clk wizard - we want to use as variable only what we may want to change